### PR TITLE
CreateFunctionUrlConfig

### PR DESCRIPTION
**Pull request by saakovv**
### Summary of the Pull Request

This rule now tracks the creation of the Lambda function URL configuration, which can be used by attackers to gain unauthorized access to Lambda functions and their privileges.

### Example Log Event

```
{ [-]
   awsCloud_account_ID: XXX
   awsRegion: eu-central
   cwlGroupname: aws-controltower/CloudTrailLogs
   eventCategory: Management
   eventID: 6e683fe2
   eventName: CreateFunctionUrlConfig
   eventSource: lambda.amazonaws.com
   eventTime: 2024-09-12T08:50:11Z
   eventType: AwsApiCall
   eventVersion: 1.08
   managementEvent: true
   readOnly: false
   recipientAccountId: XXX
   requestID: db215d0a
   requestParameters: { [-]
     authType: NONE
     cors: { [-]
     }
     functionName: XXX
     invokeMode: BUFFERED
   }
   responseElements: { [-]
     authType: NONE
     creationTime: 2024-09-12T08:50:11.556353Z
     functionArn: arn:aws:lambda:eu-central-XXX:function:XXX
     functionUrl: https://XXX
     invokeMode: BUFFERED
   }
   sessionCredentialFromConsole: true
   sourceIPAddress: XXX
   tlsDetails: { [-]
     cipherSuite: TLS_AES_128_GCM_SHA256
     clientProvidedHostHeader: lambda.eu-central.amazonaws.com
     tlsVersion: TLSv1.3
   }
   userAgent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36
   userIdentity: { [-]
     accessKeyId: XXX
     accountId: XXX
     arn: arn:aws:iam::XXX:user/XXX
     principalId: XXX
     sessionContext: { [-]
       attributes: { [-]
         creationDate: 2024-09-12T08:46:54Z
         mfaAuthenticated: false
       }
       sessionIssuer: { [-]
       }
       webIdFederationData: { [-]
       }
     }
     type: IAMUser
     userName: XXX
   }
}
```

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/)

### CODE REVIEW

File: rules/cloud/aws/cloudtrail/aws_lambda_function_url.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: AWS Created Malicious Lambda Function URL Configuration
+id: ec541962-c05a-4420-b9ea-84de072d18f4
+status: experimental
+description: |
+  Detects when a user creates a Lambda function URL configuration, which could be used to expose the function to the internet and potentially allow unauthorized access to the function's IAM role for AWS API calls.
+  This could give an adversary access to the privileges associated with the Lambda service role that is attached to that function.
+references:
+    - https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunctionUrlConfig.html
```
Comment (frack113) [inline rules/cloud/aws/cloudtrail/aws_lambda_function_url.yml:8]:
```suggestion
    - https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunctionUrlConfig.html
    - https://cloud.hacktricks.xyz/pentesting-cloud/aws-security/aws-privilege-escalation/aws-lambda-privesc
    - https://www.wiz.io/blog/how-to-set-secure-defaults-on-aws
```
Add some ref for better understanding

**Review by frack113 — COMMENTED** [review]
LGTM

**Review by saakovv — COMMENTED** [review]
added references

**Review by frack113 — APPROVED** [review]

**Comment by saakovv** [comment]
@frack113 Hi! Do you need more information from me?

**Comment by frack113** [comment]
> @frack113 Hi! Do you need more information from me?

No, thanks.
I have approve it.
I have put the labeks `2nd Review Needed`  for another menber take a look at.

**Comment by nasbench** [comment]
> > @frack113 Hi! Do you need more information from me?
> 
> No, thanks.
> I have approve it.
> I have put the labeks `2nd Review Needed`  for another menber take a look at.
> 

Aka me 😁
Would still need to check fp rate with some data I have and review. Hang in tight it's coming.

**Pull request merged**